enterprise/models.py

```diff
@@ -1683,7 +1683,9 @@ def is_active(self):
         """
         Returns True iff this enrollment is currently active.
         """
-        return self.course_enrollment.is_active
+        if self.course_enrollment:
+            return self.course_enrollment.is_active
+        return None
```

Comment:
Wouldnt you return False? Should it be none?

Reply:
IMHO, it should be `None` so that we can represent the nuance of "there isn't a student.CourseEnrollment associated with this Enterprise CE."